Parsers/Logstash/input-jsonfile-output-loganalytics.conf

```diff
@@ -0,0 +1,29 @@
+// Logstash configuration file to ingest custome json file
+// Last Updated Date: Nov 5, 2019
+//
+// Usage Instructions:
+// Follow the wiki article for installation and usage Instructions.
+// https://github.com/Azure/Azure-Sentinel/wiki/Ingest-Custom-Logs-LogStash
+//
+// Parser Notes:
+// 1. Make sure the log type name is specified as alpha characters- no digits or special characters allowed.
```

Comment:
Same thought here, remove the 1. and hyphen

Reply:
Corrected